themes/classic/modules/ps_shoppingcart/modal.tpl

```diff
@@ -59,9 +59,9 @@
                 <p class="product-price">{$product.price}</p>
```

Comment:
@PrestaShop/qa-automation Are these modifications copied to the ps_shopppingcart module ?

Reply:
Yes, please